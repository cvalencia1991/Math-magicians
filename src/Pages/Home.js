import React from 'react';

const Home = () => (
  <div>
    <h2>Welcome to our page!</h2>
    <div>
      Lorem ipsum dolor sit amet consectetur, adipisicing elit.
      Quis aliquid quae amet quisquam voluptates magni in necessitatibus molestiae animi a.
      Quis velit quia facere accusantium sed animi, nobis explicabo ratione!
    </div>
    <div>
      Lorem ipsum dolor sit amet consectetur, adipisicing elit.
      Quis aliquid quae amet quisquam voluptates magni in necessitatibus molestiae animi a.
      Quis velit quia facere accusantium sed animi, nobis explicabo ratione!
    </div>
  </div>
);
export default Home;
