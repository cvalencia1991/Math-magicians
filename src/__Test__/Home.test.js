import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Home from '../Pages/Home';

describe('Testing Home', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
