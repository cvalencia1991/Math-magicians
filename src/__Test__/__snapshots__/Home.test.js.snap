// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Home renders correctly 1`] = `
<div>
  <h2>
    Welcome to our page!
  </h2>
  <div
    className="infolorem"
  >
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quis aliquid quae amet quisquam voluptates magni in necessitatibus molestiae animi a. Quis velit quia facere accusantium sed animi, nobis explicabo ratione!
  </div>
  <div
    className="infolorem"
  >
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quis aliquid quae amet quisquam voluptates magni in necessitatibus molestiae animi a. Quis velit quia facere accusantium sed animi, nobis explicabo ratione!
  </div>
</div>
`;
